magneto_lib/magneto.h

```diff
@@ -3,10 +3,12 @@
 
 namespace cr {
 namespace magneto {
+namespace chessary {
 
 // Defines library API
 
 }  // namespace magneto
```

Comment:
The order of the comments should be the backwards of the declarations, first the `chessary` namespace closes and then the `magneto`.

Reply:
Done.